news-TS/src/components/controller/appLoader.ts

```diff
@@ -0,0 +1,11 @@
+import Loader from './loader';
+
+class AppLoader extends Loader {
+    constructor() {
+        super('https://nodenews.herokuapp.com/', {
+            apiKey: '7af2c632159747118078af9085bce66f', // получите свой ключ https://newsapi.org/
```

Comment:
I would move it to some sort of `.env` file. It is not recommended to store secrects right in the code because of security reasons